web/src/main/java/com/graphhopper/http/cli/ExportCommand.java

```diff
@@ -187,13 +190,19 @@ private static void writeStreetEdgesCsv(GraphHopper configuredGraphHopper,
                     // Copy R5's logic; filter out edges with unwanted highway tags and negative OSM IDs
                     // todo: do negative OSM ids happen in GH? This might have been R5-specific
                     if (!HIGHWAY_FILTER_TAGS.contains(highwayTag) && osmId >= 0) {
-                        // Print line for each edge direction
-                        printer.printRecord(forwardStableEdgeId, startVertex, endVertex,
-                                startLat, startLon, endLat, endLon, geometryString, streetName,
-                                distanceMillimeters, osmId, speedcms, forwardFlags, forwardLanes, highwayTag);
-                        printer.printRecord(backwardStableEdgeId, endVertex, startVertex,
-                                endLat, endLon, startLat, startLon, geometryString, streetName,
-                                distanceMillimeters, osmId, speedcms, backwardFlags, backwardLanes, highwayTag);
+                        // Print line for each edge direction, if edge is accessible.
+                        // Inaccessible edges have no flags set; flags are stored as stringified lists,
+                        // so innaccessible edges will have a flag equal to "[]", the empty list's toString()
+                        if (!forwardFlags.equals(Lists.newArrayList().toString())) {
```

Comment:
Not sure how big of a difference it makes in terms of memory/speed in Java, but would it make sense to initialize `Lists.newArrayList().toString()` ahead of time rather than when checking each edge?

Reply:
It likely will have almost no impact at all, but might still be good to change...I was wavering between checking against `Lists.newArrayList().toString()` and just `[]`; the latter felt like it was too unclear, but given that I've also added a big comment explaining this logic, I think I might as well go with `[]`.